test/Spaces/distributed/ddss2.jl

```diff
@@ -65,13 +65,14 @@ end
     yx = copy(y0)
     y2 = deepcopy(y0)
 
-    Spaces.weighted_dss!(y0)
-    p = @allocated Spaces.weighted_dss!(y0)
-    @test_broken p == 0
+    dss_buffer = Spaces.create_ghost_buffer(y0)
+    Spaces.weighted_dss!(y0, dss_buffer)
+    Spaces.weighted_dss!(y0, dss_buffer)
```

Comment:
```suggestion
```
If we're going to delete the broken allocation test for `weighted_dss`, then I think we can remove the duplicate call, since that is only needed for the allocation test (to remove compile-time allocations)